<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cookieWindow</class>
 <widget class="QMainWindow" name="cookieWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dooble Web Browser: Cookies</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QTreeView" name="cookies">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="dgenericsearchwidget" name="searchLineEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="deletePushButton">
        <property name="text">
         <string>&amp;Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteAllPushButton">
        <property name="text">
         <string>Delete &amp;All</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::MenuButtonPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="text">
         <string>&amp;Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>dgenericsearchwidget</class>
   <extends>QLineEdit</extends>
   <header>dgenericsearchwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cookies</tabstop>
  <tabstop>searchLineEdit</tabstop>
  <tabstop>deletePushButton</tabstop>
  <tabstop>deleteAllPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
